gallery/src/main.scss

```diff
@@ -81,3 +81,12 @@
     display: none;
   }
 }
+
+#temporary-navigation-sidebar-example {
```

Comment:
Are tree views in the sidebar specifically spec'd to look like this or is this just some throwaway content that you made up?  If the former, I think we need to work out a more built-in way of styling them.  Probably something similar to what `.nx-tree-view` does withing `.nx-page-sidebar` as seen at the bottom of `NxTreeView.scss`.

Reply:
> is this just some throwaway content that you made up? 

Yes